.github/workflows/image-scan.yml

```diff
@@ -203,99 +261,350 @@ jobs:
           echo "## <a name=\"scan-results\"></a>Aqua scan results" >> $GITHUB_STEP_SUMMARY
           summaryForScanner "$aquaResultFiles" "aqua"
 
-      - name: Build PR comment
-        if: github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'synchronize')
-        id: build-pr-comment
-        shell: bash
+  setup-target:
+    needs: [ setup, evaluate-results ]
+    if: needs.evaluate-results.outputs.vulnerable == 1
+    runs-on: ubuntu-latest
+    outputs:	
+      imagesJson: ${{ steps.setup.outputs.imagesJson }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.base.ref }}
```

Comment:
How does the target scanning work after the PR is merged? Do we still have `github.event.pull_request.base.ref` filled in that case? I mean if the workflow is triggered on:
```
on:
  push:
    branches:
    - master
```

Reply:
Right... If the workflow is ran outside of a PR where there isn't a target branch, it should skip the second half of the workflow.

I added a condition to the `setup-target` job so that it only runs for PRs. If that job is skipped, all following jobs that depend on it will be skipped as well.